validator/build.py

```diff
@@ -164,11 +164,11 @@ def GenValidatorProtoascii(out_dir):
   assert re.match(r'^[a-zA-Z_\-0-9]+$', out_dir), 'bad out_dir: %s' % out_dir
 
   protoascii_segments = [open('validator-main.protoascii').read()]
-  extensions = glob.glob('extensions/*/0.1/validator-*.protoascii')
+  extensions = glob.glob('extensions/*/(0.1|1.0)/validator-*.protoascii')
```

Comment:
That's why I want to use config from that file. All use 0.1 by default. The only one that needs `0.1|1.0` is `amp-sidebar`. 

I think `0.1|1.0`doesn't make sense to upgraded elements.  for example `extensions/amp-sticky-ad/0.1/validator-*.protoascii` exists, but the file is deprecated and should be built/tested no more.